decidim-initiatives/config/locales/ca.yml

```diff
@@ -54,6 +54,7 @@ ca:
         initiatives:
           fields:
             created_at: Creat a
+            id: ID
```

Comment:
```suggestion
```

We don't need to add translations at our PRs, everything is done through Crowdin (otherwise we'll have merge conflicts later!)